llvm.opam

```diff
@@ -4,7 +4,7 @@ authors: [
   "whitequark <whitequark@whitequark.org>"
```

Comment:
Note that while whitequark did contribute significant additions, AFAIU Gordon Henriksen was the original author.

Reply:
Fixed in https://github.com/kit-ty-kate/llvm-dune/commit/b9d81c73590bd1d05c2f47eb6d1e37ffb68ad581